src/luascript.cpp

```diff
@@ -11912,10 +11912,10 @@ int LuaScriptInterface::luaMonsterTypeGetAttackList(lua_State* L)
 		return 1;
 	}
 
-	lua_createtable(L, monsterType->spellAttackList.size(), 0);
+	lua_createtable(L, monsterType->attackSpells.size(), 0);
 
-	int index = 0;
-	for (const auto& spellBlock : monsterType->spellAttackList) {
+	int32_t index = 0;
```

Comment:
BTW this occurs in other places in luascript.cpp as well (e.g. in setMetatable()).